static/js/formatters-internal.js

```diff
@@ -522,6 +523,28 @@ export function price(fieldValue = {}, locale) {
   return price.toLocaleString(localeForFormatting, { style: 'currency', currency: currencyCode });
 }
 
+/**
+ * Returns a localized price range string for the given price range ($-$$$$) and country code (ISO format)
+ * @param {Object} pricerange The price range from LiveAPI entity
+ * @param {string} countrycode The country code from LiveAPI entity (e.g. profile.address.countryCode)
+ * @return {string} The price range with correct currency symbol formatting according to country code
+ */
+export function priceRange(priceRange, countryCode) {
+  if (!priceRange || !countryCode) {
+    console.warn(`No price range or country code given.`);
+    return '';
+  }
+  const currencyCode = LocaleCurrency.getCurrency(countryCode);
+  if (currencyCode) {
+    const currencySymbol = getSymbolFromCurrency(currencyCode);
+    if (currencySymbol) {
+      return Array(priceRange.length+1).join(currencySymbol); 
+    }
+  }
+  console.warn(`Unable to determine currency symbol from ISO country code ${countryCode}.`);
+  return '';
```

Comment:
In this case, would it be better to simply return the original price range, non-localized?

Reply:
I thought it might be a bit misleading to have default dollar signs on international locations if we can't localized it so I left it blank. But I can return the original price range if you think that's fine!

Reply:
My first inclination is that we return the original price range. That way, there's at least something displayed. The formatter is then strictly better than just plopping the original price range on the page. Let's go with that approach for now and see if we get QA feedback.

Reply:
got it. Updated!